src/lt/plugins/error_proposal.cljs

```diff
@@ -0,0 +1,89 @@
+(ns lt.plugins.error-proposal
```

Comment:
I think a better namespace would be lt.plugins.clojure.eval or lt.plugins.clojure.collapsible-exception if we want to be very specific.